tests/sentry/utils/suspect_resolutions/test_metric_correlation.py

```diff
@@ -0,0 +1,230 @@
+import random
+from unittest import mock
+
+from django.utils import timezone
+
+from sentry.models import GroupStatus
+from sentry.testutils import TestCase
+from sentry.utils.suspect_resolutions.metric_correlation import (
+    calculate_pearson_correlation_coefficient,
+    is_issue_error_rate_correlated,
+)
+
+START = 1656393600
+END = 1656396780
+WINDOW = 60
+
+
+def generate_issue_error_series(
+    start_timestamp, end_timestamp, data_fill, empty_fill=0, time_window=60
+):
+    return data_fill + [(t, empty_fill) for t in range(start_timestamp, end_timestamp, time_window)]
```

Comment:
This seems to be only used for metric-correlation related tests. Consider creating a `MetricCorrelationTestCase` class that inherits from `TestCase` which has this information as attributes. 

Is there any way we can avoid hard-coding the `START` and `END` timestamps? What's stopping us from using:
```
now = datetime.datetime.now()
start = now - datetime.timedelta(hours=1)
end = now
START=int(start.timestamp())
END=int(now.timestamp())
```

The point is we wanna avoid hard-coding constants in tests; sometimes its unavoidable, but in this case it doesn't look like the start and end needs to be fixed in place.